charts/stable/plex/values.yaml

```diff
@@ -1,371 +1,112 @@
-# Default values
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
-
-# The Image to use for PLEX
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
 
 image:
-  repository: plexinc/pms-docker
```

Comment:
Are we sure we want to move away from the 'official' plex image here?

Reply:
I am! People can use to use the upstream image but will lose out on some custom configuration. Someday we will update the other charts to use our images. We're getting better at documention :)